internal/utils/wallet.go

```diff
@@ -0,0 +1,12 @@
+package utils
```

Comment:
@tchardin here is the global NewKey function that already encapsulate the type of key I was talking on my last comment

Reply:
I might be too paranoid though 😁, but if you like it like that I will use this function in all the tests as well